compiler/semantic/proc.go

```diff
@@ -208,20 +208,24 @@ func semSequential(ctx context.Context, scope *Scope, seq *ast.Sequential, adapt
 	if seq == nil {
 		return nil, nil
 	}
+	scope.Enter()
+	defer scope.Exit()
+	consts, err := semConsts(scope, seq.Consts)
+	if err != nil {
+		return nil, err
+	}
 	var ops []dag.Op
 	for _, p := range seq.Procs {
-		if isConst(p) {
-			continue
-		}
 		converted, err := semProc(ctx, scope, p, adaptor, head)
 		if err != nil {
 			return nil, err
 		}
 		ops = append(ops, converted)
 	}
 	return &dag.Sequential{
-		Kind: "Sequential",
-		Ops:  ops,
+		Kind:   "Sequential",
+		Ops:    ops,
+		Consts: consts,
```

Comment:
Nit: Follow field definition order.
```suggestion
		Consts: consts,
		Ops:    ops,
```